### DISCUSSION TITLE
Flake for wezterm, what advantages?

### DISCUSSION THREAD

**Discussion by refaelsh** [wezterm/wezterm#6204] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Xmonad

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I use NixOS, btw.
I noticed there is a flake available for Wezterm.
I dont see that this flake exposes a NixOS module for wezterm configuration or something else.
So, what other advantage does this flake provides?
Besides pinning of a wezterm version in a lock file.

**Comment by nekowinston** [comment]
running the nightly build.

**Reply by refaelsh** [reply]
Is that the only advantage?

**Reply by nekowinston** [reply, marked as answer]
yes, there's already a [home-manager module](https://home-manager-options.extranix.com/?query=programs.wezterm&release=release-24.05) to configure wezterm, and you can use `programs.wezterm.package` to set it to use this repo's nightly overlay.

**Reply by ytubs** [reply]
Hey @nekowinston, could you expand on how to set home-manager to pick the nightly build? what is the value that needs to be assigned to `package` ?
